<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateTimeEdit" name="dt_edit_from">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>7</month>
     <day>11</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd-MM-yyyy H:mm</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dt_edit_to">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>7</month>
     <day>13</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd-MM-yyyy H:mm</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_from">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_to">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_device">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>РОСА К-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Тест Студии</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="download_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Device</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
